### PR TITLE
MINOR: KIP-229 Follow-up

### PR DESCRIPTION
**Pull request by vahidhashemian**
This PR includes
* Minor fix to command line output message
* Relevant documentation update

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-test-coverage/214/

File: docs/ops.html [hunk 1]
```diff
@@ -162,9 +162,8 @@ <h4><a id="basic_ops_consumer_lag" href="#basic_ops_consumer_lag">Checking consu
 
   <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing Consumer Groups</a></h4>
 
-  With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. Note that deletion is only available when the group metadata is stored in
-  ZooKeeper. When using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
-  the broker handles coordination of partition handling and rebalance), the group is deleted when the last committed offset for that group expires.
+  With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. When using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
+  the broker handles coordination of partition handling and rebalance), the group can be deleted manually, or automatically when the last committed offset for that group expires.
```
Comment (hachikuji) [inline docs/ops.html]: nit: break line

Also, maybe it is better to mention up front that the group must be empty?

**Review by hachikuji — COMMENTED** [review]
Thanks for updating the docs. Left one small comment

File: docs/ops.html (hunk 1 above)
Comment (vahidhashemian) [reply docs/ops.html]: Thanks for the feedback. The new commit should fix these.

**Review by hachikuji — APPROVED** [review]
LGTM

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-test-coverage/282/

**Pull request merged**